debian/changelog

```diff
@@ -1,3 +1,9 @@
+pg-auto-failover (1.4.2-1) stable; urgency=low
+
+  * Official 1.4.2 release of pg_auto_failover
```

Comment:
```suggestion
  * Official 1.4.2 release of pg_auto_failover
  * Adds the missing dependency postgresqlXX-contrib
```

Reply:
We will not release for  debiam so I think we don't need to change the changelog